templates/_base/layout.html.twig

```diff
@@ -30,7 +30,7 @@
             site-name="{% if config is defined %}{{ config.get('general/sitename') }}{% endif %}"
             :menu="{{ sidebarmenu() }}"
             user="{{ user|default }}"
-            :labels="{{ labels|json_decode }}"
+            :labels="{{ labels }}"
```

Comment:
Actually, what are labels?

Reply:
See a few lines above it.. It injects labels provided by Bolt, in the correct locale. 

```
{% set labels = jsonlabels(['about.bolt_documentation', 'action.visit_site', 'action.create_new', ['general.greeting', {'%name%': user_display_name}], 'action.logout', 'action.edit_profile']) %}
```

Reply:
so it is same as this?
```
[
	'about.bolt_documentation'|trans, 
	'action.visit_site'|trans, 
	'action.create_new'|trans, 
	'general.greeting'|trans({'%name%': user_display_name}), 
	'action.logout', 
	'action.edit_profile'
]|json_encode
```

Reply:
Yeah, pretty much.

Reply:
so function can be removed or at least renamed? there is nothing special about labels